databricks_cli/unity_catalog/cred_cli.py

```diff
@@ -0,0 +1,247 @@
+# Databricks CLI
+# Copyright 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from databricks_cli.click_types import DacIdClickType, JsonClickType, MetastoreIdClickType
+from databricks_cli.configure.config import provide_api_client, profile_option, debug_option
+from databricks_cli.unity_catalog.api import UnityCatalogApi
+from databricks_cli.unity_catalog.utils import mc_pretty_format
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, json_cli_base
+
+
+#############  Storage Credential Commands  ############
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Create storage credential.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/unity-catalog/storage-credentials'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_credential_cli(api_client, json_file, json):
+    """
+    Create new storage credential.
+
+    Calls the 'createStorageCredential' RPC endpoint of the Unity Catalog service.
+    The specification for the request json can be found at <insert doc link here>.
```

Comment:
replace all `insert doc link here` with public docs

Reply:
...when they exist. :)

Reply:
They all say `The public specification for the JSON request is in development.` in #471 .